src/app/onion/learning-object-builder/components/editor-action-panel/change-status-modal/change-status-modal.component.ts

```diff
@@ -1,30 +1,98 @@
-import {Component, EventEmitter, Input, Output} from '@angular/core';
-import {LearningObject} from '@cyber4all/clark-entity';
-import {BUILDER_ACTIONS, BuilderStore} from '../../../builder-store.service';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  ChangeDetectorRef
+} from '@angular/core';
+import { LearningObject } from '@cyber4all/clark-entity';
+import { BUILDER_ACTIONS, BuilderStore } from '../../../builder-store.service';
+import {
+  trigger,
+  transition,
+  style,
+  animate,
+} from '@angular/animations';
+import { ChangelogService } from 'app/core/changelog.service';
 
 @Component({
   selector: 'clark-change-status-modal',
   templateUrl: './change-status-modal.component.html',
-  styleUrls: ['./change-status-modal.component.scss']
+  styleUrls: ['./change-status-modal.component.scss'],
+  animations: [
+    trigger('carousel', [
```

Comment:
How would we feel about exporting the animations array from another file, perhaps named `clark-change-status-modal.animations.ts` (or something similar)? I've seen this in a few other projects before.

I kind of like it because it keeps the decorator less cluttered. Thoughts?